packages/esm-form-entry-app/src/app/types/index.ts

```diff
@@ -260,7 +260,15 @@ export interface Concept {
   display: string;
 }
 
+export interface Visit {
+  display?: string,
+  startDatetime: string,
+  stopDatetime?: string,
+  uuid: string,
+}
+
 export interface Encounter {
+  visit: Visit;
```

Comment:
Also, this should be further down or at least after the `uuid` property.

```suggestion
  visit?: Visit;
```

Reply:
Corrected